formatters/json_formatter.js

```diff
@@ -0,0 +1,15 @@
+// Copyright 2017 TODO Group. All rights reserved.
+// Licensed under the Apache License, Version 2.0.
+
+class JsonFormatter {
+  format (rule, message, level) {
+    let output = {}
```

Comment:
How about inlining the initialization like

    let output = {
        rule: rule.id,
        ...

?

Reply:
Good call. Will do.

Reply:
Thanks for the quick response, @sschuberth! I've made that change.